### DISCUSSION TITLE
Azure Virtual Desktop Landing Zone Accelerator

### DISCUSSION THREAD

**Discussion by matthewjlevy** [Azure/Enterprise-Scale#937] [Q&A]
Hi ESLZ folks. Love your work firstly.

Secondly, I have been informed by an AVD Masterclass by Vivek Massy (Microsoft) that there is an Azure Virtual Desktop Landing Zone Accelerator. I can't seem to find it in this repo. Would you have any idea of when it will be available for Portal deployment?

Below is the slide from Vivek's presentation mentioning the accelerator:

![avdlza](https://user-images.githubusercontent.com/27995471/157888690-b360589e-11a2-4d36-ae7d-c36310d92b87.jpg)

**Comment by jtracey93** [comment]
@danycontre Can you help here?

**Reply by FallenHoot** [reply]
Old topic, but it can be found here:
https://github.com/Azure/avdaccelerator

It doesn't align with ALZ as of today.
